client/mainmenu/CCampaignScreen.cpp

```diff
@@ -129,7 +129,8 @@ void CCampaignScreen::CCampaignButton::show(SDL_Surface * to)
 	}
 	else if(CCS->videoh->fname == video) // When you got out of the bounds of the button then close the video
 	{
-		CCS->videoh->close();
+		// Don't close the video to prevent crash, may lead to memory leak
+		// CCS->videoh->close();
```

Comment:
Video issue is a known problem. Would be great if you fix it. First of all where exactly it crashes? What thread is it? What is call stack? It crashes on Windows but not on Linux?